src/Schema/Schema.php

```diff
@@ -58,17 +58,17 @@ class Schema implements SchemaInterface
     /**
      * @var TypeInterface|null
      */
-    protected $query;
+    protected $queryType;
 
     /**
      * @var TypeInterface|null
      */
-    protected $mutation;
+    protected $mutationType;
 
     /**
      * @var TypeInterface|null
      */
-    protected $subscription;
+    protected $subscriptionType;
```

Comment:
@crisu83 One thing I want to ask is can we return `ObjectType` instead of `TypeInterface`. Since they cannot be any thing else. And this is only for internal api.

Reply:
We cannot, I cannot recall why, but there was a reason for why it has to be `TypeInterface|null`.

Reply:
I think it's probably because of using container?